crates/iota/src/validator_commands.rs

```diff
@@ -538,26 +538,23 @@ impl IotaValidatorCommand {
                     validator_address,
                     print_unsigned_transaction_only,
                 )?;
-                // Make sure the address is a validator
+
+                // in the Move code (`bridge::committee::register`) we use
+                // `active_validator_addresses` to get the list of active validators,
+                // so here we allow all active validators to register as members of the
+                // `BridgeCommittee`.
```

Comment:
But is this correct to have different members of the bridge committee and the IOTA committee?

Reply:
Also for 
> So I assume that this was changed back be active_validators in the end, or was this missed somehow?
This is a good question. Because 
```
let active_validators = iota_system_state.iter_committee_members();
```` on Line 45 already makes the active_validators be `Iterator`, so here we don't need to use `into_iter()` again.

Reply:
Uh, I guess the move code needs to be changed then, because as far as I understand, the bridge should be run by the same committee as the consensus.

Reply:
Got it. Modified the comments and codes at [94c3ce1](https://github.com/iotaledger/iota/pull/6015/commits/94c3ce1ce77b08888557dd77e26b333d7cd8abea).
Note that the corresponding modification of Move code should be addressed in issue [6109](https://github.com/iotaledger/iota/issues/6109)